### PR TITLE
fern: tau Y/Z absolute upscaling (preserved ratio) — 4-ep screen

### PR DESCRIPTION
**Pull request by morganmcg1**
# Tau Y/Z Absolute Upscaling (Preserved Ratio) — 4-ep Screen

## Hypothesis

τ_y and τ_z are the dominant performance bottleneck in every 4-ep run on the current SOTA stack. The SOTA config uses `--tau-y-loss-weight 1.5 --tau-z-loss-weight 2.0` (ratio 0.75). These weights have never been scaled up in absolute magnitude — only τ_y was increased by askeladd in PR #571, and askeladd's in-flight PR #849 tests a **ratio flip** (τ_y priority over τ_z).

This PR tests a fundamentally different axis: **raising BOTH τ_y and τ_z weights proportionally**, preserving approximately the same ratio while increasing the absolute gradient signal for both weak channels. The frieren #847 per-channel data showed τ_y≈10%, τ_z≈11% even at EP4 on the SOTA config — frieren explicitly flagged this as a follow-up in their PR comments.

This is **orthogonal to PR #849** (which tests ratio flip while keeping τ_z fixed or lower) and **orthogonal to PR #853** (which tests model dropout regularization).

## Arms

- **Arm A**: `--tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5` (ratio 0.80 — moderate absolute step up)
- **Arm B**: `--tau-y-loss-weight 2.5 --tau-z-loss-weight 3.0` (ratio 0.83 — more aggressive upscaling)

All other hyperparameters match the current SOTA exactly (PR #592, W&B run `4k25s25e`).

## Current SOTA Baseline

**Single-model SOTA: PR #592 (alphonse, depth-L5)**
- val_abupt = **6.5985%** (EP4, 4-ep screen)
- test_abupt = 7.9915%
- Val per-channel: surface_pressure=4.3322%, volume_pressure=3.9456%, tau_x=6.5420%, tau_y=8.3631%, tau_z=9.8099%

**Gate to beat (4-ep screen):** val_abupt < **6.5985%**

Kill gates (4-ep screen):
- EP1 < 30% (warmup epoch, gate relaxed)
- EP2 < 16%
- EP3 < 8%
- EP4 ≤ 6.5985%

## Instructions

Run both arms as 4-ep screens. Use `--wandb-group fern-tau-yz-upscale` to group both runs.

**Arm A** (`tau_y=2.0, tau_z=2.5`):

```bash
cd target/ && torchrun --standalone --nproc_per_node=8 train.py \
  --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 \
  --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 \
  --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 \
  --pos-encoding-mode string_separable --use-qk-norm \
  --rff-num-features 16 --rff-init-sigmas "0.25,0.5,1.0,2.0,4.0" \
  --lr-cosine-t-max 13 --epochs 4 \
  --vol-points-schedule "0:16384:1:32768:2:49152:3:65536" \
  --no-compile-model \
  --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
```

**Arm B** (`tau_y=2.5, tau_z=3.0`):

```bash
cd target/ && torchrun --standalone --nproc_per_node=8 train.py \
  --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 \
  --tau-y-loss-weight 2.5 --tau-z-loss-weight 3.0 --surface-loss-weight 2.0 \
  --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 \
  --pos-encoding-mode string_separable --use-qk-norm \
  --rff-num-features 16 --rff-init-sigmas "0.25,0.5,1.0,2.0,4.0" \
  --lr-cosine-t-max 13 --epochs 4 \
  --vol-points-schedule "0:16384:1:32768:2:49152:3:65536" \
  --no-compile-model \
  --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.5-tauz3.0-4ep
```

## Reporting

Report the EP1–EP4 val_abupt for each arm, plus the per-channel val metrics at EP4. Apply the 4-ep kill gates above. If Arm A or B passes all 4 gates (EP4 ≤ 6.5985%), mark this PR as ready for review.

If an arm fails a kill gate early, terminate that arm and report the result with the gate failure noted. If both arms fail, close this PR with results.

Include the W&B run IDs in your report comment.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Fern — this tay screen (tau Y/Z absolute loss upscaling) has had no activity. I understand you're currently running the long-run PR #831 which is showing excellent progress (best 6.5644% @ EP23 — wave leader). However this tay screen is only a 4-epoch run (~5-6 hours total) and represents a separate GPU slot. Please pick up PR #856 as soon as your current GPU situation allows and post a launch confirmation with your W&B run ID. The training command is in the PR body.

**Comment by morganmcg1** [comment]
ADVISOR: Second escalation — PR #856 (tau Y/Z absolute loss upscaling) still has no launch confirmation. This is now a stale tay screen slot. If you are unable to start this run, please let me know so I can reassign. Otherwise, please launch it immediately and post your W&B run ID. The training command is in the PR body. This experiment tests a physically motivated idea (tau upscaling on Y/Z axes) and we need the data — please do not let this sit idle any longer.

**Comment by morganmcg1** [comment]
ADVISOR: Third escalation — PR #856 (tau Y/Z absolute loss upscaling) has now had ZERO activity for over 30 minutes since assignment. This is a critical tay screen slot that has been sitting idle. 

If you cannot launch this run for any reason (conflicting GPU allocation, technical issues, etc.) please say so NOW in a comment so I can reassign this slot to another student. Otherwise, please launch it immediately.

Command is in the PR body. Run it and post the W&B run ID in the next 15 minutes or I will reassign this slot.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #856 because ADVISOR: Closing this PR due to zero student activity after three escalation warnings spanning 30+ minutes. This tay screen slot is being reassigned. The tau Y/Z absolute loss upscaling hypothesis is worth testing — it will be reassigned to another student..

**Comment by morganmcg1** [comment]
STUDENT-WATCHDOG: stopping this student invocation on senpai-drivaerml-ddp8-fern-d4b8ddf77-84rfr.

Reason: assignment changed from #856 to none for 1778249142s while train.py is active
Original assignment: #856
Current assignment: none

Active training processes before termination:

```
1101344 1101342    6422  0.2  0.0 pt_elastic      /usr/bin/python3 /usr/local/bin/torchrun --standalone --nproc_per_node=8 train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101426 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101427 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101428 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101429 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101430 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101431 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101432 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
1101433 1101344    6421  100  0.2 python3         /usr/bin/python3 -u train.py --agent fern --optimizer lion --lr 9e-5 --weight-decay 5e-4 --tau-y-loss-weight 2.0 --tau-z-loss-weight 2.5 --surface-loss-weight 2.0 --ema-decay 0.999 --grad-clip-norm 0.5 --lr-warmup-epochs 1 --pos-encoding-mode string_separable --use-qk-norm --rff-num-features 16 --rff-init-sigmas 0.25,0.5,1.0,2.0,4.0 --lr-cosine-t-max 13 --epochs 4 --vol-points-schedule 0:16384:1:32768:2:49152:3:65536 --no-compile-model --model-layers 5 --model-hidden-dim 512 --model-heads 4 --model-slices 128 --batch-size 4 --validation-every 1 --train-surface-points 65536 --eval-surface-points 65536 --train-volume-points 65536 --eval-volume-points 65536 --wandb-group fern-tau-yz-upscale --wandb-name fern/tauy2.0-tauz2.5-4ep
```

The watchdog only takes this action after the assignment drift remains stable for 1800s, so transient GitHub/API wobbles should not trigger it.
